Dockerfile-27

```diff
@@ -1,7 +1,7 @@
 # syntax=docker/dockerfile:1
 
-ARG NEXTCLOUD_VERSION=27.1.7
-ARG ALPINE_VERSION=3.19
+ARG NEXTCLOUD_VERSION=27.1.9
+ARG ALPINE_VERSION=3.20
```

Comment:
Don't bump to alpine 3.20 yet, I need to do some testing first with my s6-overlay image.